components/operator/internal/modules/gateway/handler.go

```diff
@@ -192,6 +193,10 @@ func createCaddyfile(context modules.ServiceInstallConfiguration) string {
 		"Port":     gatewayPort,
 		"Fallback": fallback,
 		"Redirect": redirect,
+		"EnableAudit": func() bool {
```

Comment:
Should we add boolean in the configuration crd that activate or not the audit ?

Reply:
Added in Configuration.spec.services.gateway.enableAuditPlugin